x11-packages/thunderbird/0022-wayland-proxy-no-pthread-cancel.patch

```diff
@@ -0,0 +1,12 @@
+--- a/third_party/wayland-proxy/wayland-proxy.cpp
++++ b/third_party/wayland-proxy/wayland-proxy.cpp
+@@ -739,7 +739,9 @@
+   if (mThreadRunning) {
+     Info("thread is still running, terminating.\n");
+     mThreadRunning = false;
++#ifndef __TERMUX__
+     pthread_cancel(mThread);
```

Comment:
Should `wayland` be disabled as firefox does?

Reply:
Why is it disabled in `firefox`?
Just lack of testing, or is there any acute issues with it?

Reply:
https://github.com/termux/termux-packages/pull/19441#issuecomment-2000251881

Reply:
From my very rough understanding it looks like a NULL pointer got into the Wayland clipboard handling.
It would probably be good to do some re-investigating on that, I'd be happy to help out with that if possible.

Reply:
Emmm... I think it can be addressed and fixed in a seperate PR as there will not be too many wayland users in Termux.

Reply:
Yeah absolutely.
No need to hold up this PR.

Reply:
I leave up to you all to fix the remaining issues after merge